qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -273,20 +297,29 @@ def prev_result(self, *, result_cb=None):
             int(self._flags))  # type: ignore[call-overload]
         if flags & QWebEnginePage.FindBackward:
             if self._wrap_handler.prevent_wrapping(going_up=False):
+                result_cb(True)
```

Comment:
I did this because it's what I would expect if I were using the searching API as a black box. When you call `prev_result` or `next_result` and give them your callback, you expect that callback to be called when the searching is done. The fact that the wrap prevention results in no "search" actually being performed is an implementation detail from that perspective. The callback is called with `True` because a successful search should already be active when calling those functions - I think the fact that they "work" even if there isn't is kind of weird from an API perspective anyway.